packages/yoga/src/NavigationMenu/web/BottomItems/BottomItems.tsx

```diff
@@ -9,14 +9,14 @@ const StyledItemsContainer = styled.nav`
 
 type BottomItemsStyledProps = React.HTMLAttributes<HTMLUListElement> & {
```

Comment:
was not introduced here, but `z-index: ${({ $zIndex }: BottomItemsStyledProps) => $zIndex ?? 2};` should be applied to `styled.nav` instead of being applied to `styled.ul`

Reply:
Fixed.
I've also updated zIndex prop so it's exposed without the `$`, which is a implementation detail so this prop doesn't go to the DOM.

https://styled-components.com/docs/api#transient-props